tests/github-action-helper.sh

```diff
@@ -104,6 +104,28 @@ wait_for_operator_pod_to_be_ready_state_custom() {
 EOF
 }
 
+wait_for_condition() {
```

Comment:
This function is always called twice, once for pending, then for running state. How about if inside this function it checked for pending then running automatically? It really will simplify all the places this function is called.
```suggestion
wait_for_running_pod() {
```

Reply:
Updated it, But according to this above discussion it will again become a complicated one-liner https://github.com/rook/kubectl-rook-ceph/pull/48#discussion_r960734689

Reply:
Do we need to check for pending at all, or can we just wait for running?

Reply:
It may not be required, removed it